pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -170,15 +171,18 @@ public BaseBrokerRequestHandler(PinotConfiguration config, String brokerId, Brok
     _brokerTimeoutMs = config.getProperty(Broker.CONFIG_OF_BROKER_TIMEOUT_MS, Broker.DEFAULT_BROKER_TIMEOUT_MS);
     _queryResponseLimit =
         config.getProperty(Broker.CONFIG_OF_BROKER_QUERY_RESPONSE_LIMIT, Broker.DEFAULT_BROKER_QUERY_RESPONSE_LIMIT);
+    _queryMaxSerializedResponseBytes = config.getProperty(Broker.CONFIG_OF_MAX_QUERY_RESPONSE_SIZE_BYTES,
```

Comment:
Since this is a instance config that doesn't change unless restarted, I've actually added a log to indicate this value. I can add a gauge metric for sure, but do you think it'll be helpful?

Reply:
It can be useful to be used as a max. For example we use to start with Xmx = Xms, but we use the max heap value to calculate the % of heap used. We could do something like that with this metric.

Reply:
Added guage.